### ISSUE TITLE
Update database to support Gen9

### ISSUE THREAD

**Issue by tel0065** [vsoch/pokemon#10]
With the release of Pokemon Scarlet & Violet we've moved into Generation 9 with a total of 1008 Pokemon. The current database included with this project ends at Eternatus (#890) which was Sword & Shield before the DLC (Gen8). 

I saw you included a script to manually update the database for occasions such as this, however the script is throwing errors and therefor not completing. 

![image](https://user-images.githubusercontent.com/77864718/212763808-d0732361-b166-4f45-bec2-9996a0d91261.png)

**Comment by vsoch** [comment]
Thank you - I can take a look this evening! Can you copy paste the exact command you are running to the issue here for me to reproduce (so I don't need to translate from the screenshot?) Thanks!

**Comment by tel0065** [comment]
> Can you copy paste the exact command you are running to the issue here for me to reproduce (so I don't need to translate from the screenshot?)

using Python 3.11.1 

`py make_db.py`
This was per your instructions listed [here](https://github.com/vsoch/pokemon#updating-the-database)

Before running that I also installed BeautifulSoup4 and Pillow since it was giving me errors of not having those and the script utilizing them which I did by running:

`pip install beautifulsoup4`
This was per [here](https://www.crummy.com/software/BeautifulSoup/)
`pip install --upgrade Pillow`
This was per [here](https://pillow.readthedocs.io/en/latest/installation.html)

Note, I am on a Window 10 box presently. Tried both using cmd and Git Bash terminals, both being run as administrators. 

So in summary the order of commands was as follows:
```
pip install beautifulsoup4
pip install --upgrade Pillow
py make_db.py
```

Hope that helps! Thanks!

Edit: Also forgot to mention that the [pokemondb site](https://pokemondb.net/) you're using to pull the images from which you then convert to ASCII does include the rest of the [Gen8 pokemon (post Eternatus)](https://pokemondb.net/pokedex/national#gen-8) and the [Gen9 pokemon](https://pokemondb.net/pokedex/national#gen-9) already.

**Comment by vsoch** [comment]
Heyo! So I couldn't reproduce this error, but I was able to fix the install issues and also generate Gen9 from the site. Could you take a look? https://github.com/vsoch/pokemon/pull/11

Apologies for the issues you ran into - I am wincing looking at the bugs that I had. Likely after this PR looks good to you (and we merge) I'll do a bit of cleanup on the library and then a release.

**Comment by tel0065** [comment]
Appreciate it! Sorry it wasn't reproducible perhaps it's something on this machine. It's pretty old so there could be some conflicts somewhere -- I really should reformat it and start fresh. 

I looked at the changes and they look good to me.

**Comment by vsoch** [comment]
Definitely! What sticks out to me in your error message is "wget" is not a recognized as an internal or external command - so I think the issue (based on the file not found at the end) is that you don't have wget on your system, because it's (unfortunately, haha) used in the update script:

```
"wget %s -O %s --no-check-certificate --secure-protocol=TLSv1"
```
Apparently me of 7 years ago thought this was an OK thing to do (running a command from Python!) Me of today is doing facepalms. :laughing:

**Comment by vsoch** [comment]
Thanks @tel0065 ! https://pypi.org/project/pokemon/0.36/
